test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua

```diff
@@ -0,0 +1,147 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: ShowConstantTBT
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [ShowConstantTBT] SUCCESS: getting SUCCESS:Navigation.ShowConstantTBT()
+--
+-- Description:
+-- Mobile application sends valid ShowConstantTBT request and gets Navigation.ShowConstantTBT "SUCCESS" response from HMI
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+-- c. appID is currently in Background, Full or Limited HMI level
+
+-- Steps:
+-- appID requests ShowConstantTBT with valid parameters
+
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if Navigation interface is available on HMI
+-- SDL checks if ShowConstantTBT is allowed by Policies
+-- SDL checks if all parameters are allowed by Policies
+-- SDL transfers the Navigation part of request with allowed parameters to HMI
+-- SDL receives Navigation part of response from HMI with "SUCCESS" result code
+-- SDL responds with (resultCode: SUCCESS, success:true) to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Local Variables ]]
+local putFileParams = {
+  requestParams = {
+    syncFileName = 'icon.png',
+    fileType = "GRAPHIC_PNG",
+    persistentFile = false,
+    systemFile = false
+  },
+  filePath = "files/icon.png"
+}
+
+local requestParams = {
+  navigationText1 = "navigationText1",
+  navigationText2 = "navigationText2",
+  eta = "12:34",
+  totalDistance = "100miles",
+  turnIcon = {
+    value = "icon.png",
+    imageType = "DYNAMIC",
+  },
+  nextTurnIcon = {
+    value = "icon.png",
+    imageType = "DYNAMIC",
+  },
+  distanceToManeuver = 50.5,
+  distanceToManeuverScale = 100.5,
+  maneuverComplete = false,
+  softButtons = {
+    {
+      type = "BOTH",
+      text = "Close",
+      image = {
+        value = "icon.png",
+        imageType = "DYNAMIC",
+      },
+      isHighlighted = true,
+      softButtonID = 44,
+      systemAction ="DEFAULT_ACTION",
+    },
+  },
+}
+
+local responseUiParams = {
+  navigationTexts = {
```

Comment:
@GetmanetsIrina please add timeToDestination param value

Reply:
@AKalinich-Luxoft , Done in e5c425f45f8c09e764332a840527f30c5ab2c793.